snapcraft/plugins/go.py

```diff
@@ -67,18 +58,40 @@ def env(self, root):
                 '$LDFLAGS'
             ]).format(root, snapcraft.common.get_arch_triplet()) + '"',
         ]
+        return env
 
     def pull(self):
         # use -d to only download (build will happen later)
         # use -t to also get the test-deps
-        self._run(['go', 'get', '-t', '-d', self.fullname])
+        super().pull()
```

Comment:
So if the source is git://name/project, will this get the repo two times?
first pull will download it to src, and then _local_pull will download it to src/go/src?
I'm not sure if I'm missing something.